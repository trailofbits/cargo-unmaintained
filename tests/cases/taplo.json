[
  {
    "name": "adler",
    "outdated_deps": [],
    "repo_status": "Archived",
    "version": "1.0.2"
  },
  {
    "name": "dirs",
    "outdated_deps": [],
    "repo_status": "Archived",
    "version": "1.0.5"
  },
  {
    "name": "hermit-abi",
    "outdated_deps": [],
    "repo_status": "Unassociated",
    "version": "0.1.19"
  },
  {
    "name": "proc-macro-error",
    "outdated_deps": [
      {
        "name": "syn",
        "req": "^1",
        "version_latest": null,
        "version_used": "1.0.103"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "1.0.4"
  },
  {
    "name": "rust-argon2",
    "outdated_deps": [
      {
        "name": "base64",
        "req": "^0.13",
        "version_latest": null,
        "version_used": "0.13.1"
      },
      {
        "name": "blake2b_simd",
        "req": "^0.5",
        "version_latest": null,
        "version_used": "0.5.11"
      },
      {
        "name": "constant_time_eq",
        "req": "^0.1.4",
        "version_latest": null,
        "version_used": "0.1.5"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.8.3"
  },
  {
    "name": "serde_cbor",
    "outdated_deps": [],
    "repo_status": "Archived",
    "version": "0.11.2"
  },
  {
    "name": "structopt-derive",
    "outdated_deps": [
      {
        "name": "heck",
        "req": "^0.3.0",
        "version_latest": null,
        "version_used": "0.3.3"
      },
      {
        "name": "syn",
        "req": "^1",
        "version_latest": null,
        "version_used": "1.0.103"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.4.18"
  },
  {
    "name": "structopt",
    "outdated_deps": [
      {
        "name": "clap",
        "req": "^2.33",
        "version_latest": null,
        "version_used": "2.34.0"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "0.3.26"
  },
  {
    "name": "tinytemplate",
    "outdated_deps": [
      {
        "name": "criterion",
        "req": "^0.3",
        "version_latest": null,
        "version_used": "0.3.6"
      }
    ],
    "repo_status": {
      "Age": null
    },
    "version": "1.2.1"
  },
  {
    "name": "tokio-rustls",
    "outdated_deps": [],
    "repo_status": "Unassociated",
    "version": "0.23.4"
  }
]
