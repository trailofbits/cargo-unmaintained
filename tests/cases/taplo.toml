url = "https://github.com/tamasfe/taplo"
rev = "0.8.1"
use_real_github = true
