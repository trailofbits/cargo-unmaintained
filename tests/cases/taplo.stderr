Scanning 262 packages and their dependencies (pass --verbose for more information)
